### PR TITLE
Updates the Install TOC

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
Rewords the Install and Configure collection in the table of contents to Install and Upgrade, since it now hosts the upgrade instructions and rewords the Upgrade to OpenSearch collection in the TOC to Migrate to OpenSearch.

### Issues Resolved
Partially fixes https://github.com/opensearch-project/documentation-website/issues/2949; We still need to combine two files > Migrating from Elasticsearch OSS to OpenSearch / (add to previous topic - Migrating Docker clusters to OpenSearch

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _upgrade-to/upgrade-to.md [hunk 1]
```diff
@@ -98,12 +98,12 @@ If you are upgrading an Open Distro for Elasticsearch cluster, we recommend firs
...
    ```
 
-1. If you upgraded from 5.x to 6.x, [reindex]({{site.url}}{{site.baseurl}}/opensearch/reindex-data/) all indices.
+1. If you upgraded from 5.x to 6.x, [reindex]({{site.url}}{{site.baseurl}}/opensearch/reindex-data/) all indexes.
 
 1. Repeat all steps as necessary until you arrive at your desired Elasticsearch OSS version.
 
 
-## Upgrade to OpenSearch
+## MIgrate to OpenSearch
```
Comment (JeffH-AWS) [inline _upgrade-to/upgrade-to.md:106]: Accidental capital letter

**Review by JeffH-AWS — APPROVED** [review]
Looks great when I build locally. Thanks for making those changes! Only one comment, just an accidental extra capital letter

File: _upgrade-to/upgrade-to.md (hunk 1 above)
Comment (hdhalter) [reply _upgrade-to/upgrade-to.md]: Thanks for catching that!

**Review by cwillum — APPROVED** [review]
Looks good.

**Pull request merged**